js_modules/dagster-ui/packages/ui-core/src/schedules/ScheduleDetails.tsx

```diff
@@ -130,7 +130,10 @@ export const ScheduleDetails = (props: {
               </Box>
             </td>
             <td>
-              <Box flex={{direction: 'row', alignItems: 'center'}}>
+              <Box
+                flex={{direction: 'row', gap: 12, alignItems: 'center'}}
+                style={{height: '32px'}}
```

Comment:
Why are we hardcoding the height? If we want more vertical space we should use vertical padding.

Reply:
I copied the pattern from https://github.com/dagster-io/dagster/blob/2e202201629f72b6c343e42db77fa0b278a9f9a3/js_modules/dagster-ui/packages/ui-core/src/sensors/SensorDetails.tsx#L192-L194. Happy to change it though.